### DISCUSSION TITLE
myblog/dawn-of-personal-computing/

### DISCUSSION THREAD

**Discussion by giscus** [jasoneckert/jasoneckert.github.io#5] [Announcements]
# myblog/dawn-of-personal-computing/

My personal website and blog

https://jasoneckert.github.io/myblog/dawn-of-personal-computing/

**Comment by cuvtixo** [comment]
You wrote a respectable summary of the DEC machine, but I want to correct a few misconceptions. Digibarn suggests, the VT180 was only available to DEC employees, so it makes sense that neither "VT180" nor "Robin" were oficial names. It probably would have been changed if it ever reached the market. DEC didn't market a "standalone" computer until the DEC Rainbow 100, and they pulled out all the stops, It had a 8088 to run MS-DOS, and different flavors of CP/M, automatically detecting whether it was CP/M-86 (the name is deceptive, it was made to run on the 8088) or CP/M for the Z80. Also CP/M for the 8080, if one happened to have a library of software titles for it. It also could be used as a DEC VT100 terminal. Unfortunately, DEC MS-DOS wasn't very compatible with PC-DOS software, and, in addition for the price, that doomed it. One has to wonder if McLaughlin was a DEC employee or perhaps married to one. The "big picture" therefore is somewhat a distorted history. You note only Apple and IBM survived, but the DEC offering didn't even exist long enough to be recognized by anyone other than hardcore nerds(I say that as a compliment to you!)  In my mind, it emphasizes how delusion techno-utopianism, or even just tech optimism was! DEC barely got a chance to participate in the microcomputer "revolution", and honestly, the ZX81 looks like an afterthought, it doesn't blend in at all.  I'm glad you get enjoyment from this, but the bitter, disillusioned cranks like me can have quite a different reaction. This was how the road to hell was paved!!  ;P
